includes/polling/os/ptp250.inc.php

```diff
@@ -9,7 +9,34 @@
  * option) any later version.  Please see LICENSE.txt at the top level of
  * the source code distribution for details.
 */
+use LibreNMS\RRD\RrdDefinition;
 
 $version = $poll_device['sysDescr'];
 $masterSlaveMode = ucfirst(snmp_get($device, 'masterSlaveMode.0', '-Oqv', "CAMBIUM-PTP250-MIB"));
 $hardware = 'PTP 250 '. $masterSlaveMode;
+
+$txModulation = snmp_get($device, ".1.3.6.1.4.1.17713.250.5.9.0", "-Ovqn", "");
```

Comment:
I think modulation is usually a rate sensor.

Reply:
Good call, I'll merge that in. Thanks